typescript/web/src/components/members/table-actions.tsx

```diff
@@ -25,6 +27,19 @@ import { InviteMember } from "./types";
 
 const SearchIcon = chakra(IoSearch);
 
+const tableActionsForLocalWorkspace = (
+  <Alert status="info" variant="subtle">
+    <AlertIcon />
+    <Box flex={1}>
+      <AlertTitle>This workspace is private.</AlertTitle>
+      <AlertDescription>
+        Images are stored locally on your device. To collaborate and invite
+        people, switch to an online workspace.
+      </AlertDescription>
+    </Box>
+  </Alert>
+);
+
```

Comment:
@Jordanlelay ⚠️ This is not a function ⚠️ Since the PR is merged and nothing broke (nothing broke?), let's keep it for now. But it might be cleaner to keep using function components in the future.

Reply:
You're right, I just kept the things as they were.

It works because it is returned "as is", meaning that `tableActionsForLocalWorkspace` (as well as `tableActionsForOnlineWorkspace`) is of type JSX.Element which is returned by the component `TableActions`.
I will convert them to functional components to keep coherence with the rest of the code